xml/art_sle_amazon_ecs.xml

```diff
@@ -184,5 +253,46 @@
       </orderedlist>
     </sect2>
  </sect1>
+ <sect1 xml:id="caveats">
+    <title>Additional Considerations</title>
+    <para>
+      The integrated ECS service requires access to Amazon S3 to download a
+      pre-built container. This implies that if you start an ECS instance that
+      has no route to reach Amazon S3 the initialization will fail and the
+      container host instance will not be able to attach itself to your
+      cluster. Should you end up in a situation where the instance does not
+      attach itself to your <emphasis role="strong">default</emphasis>
+      cluster, you can modify your network configuration to ensure a route
+      to Amazon S3 and then simply log into your container instance as start
+      the service as follows:
+    </para>
+    <screen>$ systemctl start amazon-ecs.service</screen>
+ </sect1>
+ <sect1 xml:id="using sles containers">
+    <title>Using SUSE Linux Enterprise Server Containers</title>
+    <para>
+      SUSE Linux Enterprise Server Docker images are not available on
+      Docker Hub. However, SUSE provides pre-build docker images as
+      packages in the SLE-Module-Containers12-Updates repository. Given
+      that your launched instance has a route to the Internet and that
+      the traffic originates with an IP address assigned by EC2 the
+      instance will automatically register with an update server and
+      the SLE-Module-Containers12-Updates repository will be configured.
+      Once configured installing the docker image is simple:
+    </para>
+    <screen>$ zypper se docker-image</system>
+    <para>
+      Install the image you are interested in, the SUSE Linux Enterprise
+      Server 12 image for example:
+    </para>
+    <screen>$ zypper in sles12-docker-image</screen>
+    <para>
+      The SUSE Linux Enterprise Server EULA does not allow you to push this
```

Comment:
Not sure if we have an "EULA". Better name it just "license"? EULA sounds a bit Microsoft-ish. :grinning: